JustSaying.Logging/LoggerExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+
+namespace JustSaying.Logging
+{
+    public static class LoggerExtensions

```

Comment:
Longer term we could inline these methods, as it's just a shim for existing methods to ILogger ?